src/main/java/seedu/address/model/tag/ModuleTag.java

```diff
@@ -61,4 +93,28 @@ public String toString() {
     public int compareTo(ModuleTag otherModuleTag) {
         return tagName.compareTo(otherModuleTag.tagName);
     }
+
+    public String getEndTime() {
```

Comment:
Hmm a little strange that these methods return strings. Could possibly add the ‘AsString’ to the method names to make it clearer.

Reply:
Cos if you can, just return the correct type.